doc/DEMO_TESTNET.md

```diff
@@ -0,0 +1,327 @@
+- one cosigner noise private key and its public key
+
+You can generate bip32 key pair on bip32.org
+For a noise key pair and a cosigner bitcoin key pair you can do:
+
+`python3 -c 'import os;print(os.urandom(32).hex())'`
+
+### Docker (kinda optional)
+
+We need docker to spin up the `coordinatord`'s Postgre database.
+You can avoid using docker though, and just spin up a Postgre database yourself.
+
+## Downloading the repositories
+
+Let's start by creating a dedicated folder - this way if you don't like Revault it will be easier to erase everything :D
+
+```
+mkdir revault_tutorial
+cd revault_tutorial
+```
+
+### If you are a stakeholder:
+
+Download all the needed repositories: `cosignerd`, `revaultd`, `revault-gui`:
+
+```
+git clone -b 0.2 https://github.com/revault/cosignerd
+git clone -b 0.2 https://github.com/revault/revaultd
+git clone -b 0.2 https://github.com/revault/revault-gui
+```
+
+Then in each of the repositories:
+
+```
+cargo build
+```
+
+Binaries will be installed in `~/.cargo/bin`.
+
+### If you are a manager:
+
+Download all the needed repositories: `revaultd`, `revault-gui`:
+
+```
+git clone -b 0.2 https://github.com/revault/revaultd
+git clone -b 0.2 https://github.com/revault/revault-gui
+```
+
+Then in each of the repositories:
+
+```
+cargo build
+```
+
+## Spinning up Bitcoin Core
+
+Start a Bitcoin Core node using
+
+```
+bitcoind -testnet -daemon
+```
+
+## Starting the coordinator
+
+As we said, we need just one coordinator running, no matter how many stakeholders/managers there are.
+We'll properly update the coordinator configuration later. For now, let's just retrieve the coordinator's noise key.
+Cd into the coordinator:
+
+```
+cd coordinatord
+```
+
+Coordinatord needs a Postgre database running, we'll spin it up using docker:
+```
+docker run --rm -d -p 5432:5432 --name postgres-coordinatord -e POSTGRES_PASSWORD=revault -e POSTGRES_USER=revault -e POSTGRES_DB=coordinator_db postgres:alpine
+
+```
+
+Now duplicate the config provided:
+
+```
+cp contrib/config.toml coordinatord_config.toml
+```
+
+Make sure that the `postgres_uri` is `postgresql://revault:revault@localhost:5432/coordinator_db`.
+change the noise keys with the public noise keys of the participants.
+
+Your config file now should look like this (with your keys in it instead of those dummy keys, obviously):
+```
+daemon=true
+data_dir = "./revault_coordinatord"
+log_level = "debug"
+
+postgres_uri = "postgresql://revault:revault@localhost:5432/coordinator_db"
+
+managers = [
+    "d2deeb8398f47789e1f5118c42834031e3722817a432192b74c363fdb36cc634",
+]
+
+stakeholders = [
+    "eecd2a93f5b09b88519f38d620aa127333d1934987d18001132f07ffa3596c65",
+    "614ad96890c309b8da6915ddb9eb1135caf228120178833e70798f20e0783b16",
+]
+
+watchtowers = []
+```
+
+start the `coordinatord`:
+
+```
+~/revault_tutorial/coordinatord$
+cargo run -- --config coordinatord_config.toml
+```
+
+Then please keep note of the noise key printed at startup, it will ask
```

Comment:
nit, replace with:
```
Then please keep note of the noise key printed at startup, it will be asked while installing as the "Coordinator public noise key"
```